ui/raidboss/data/05-shb/trial/ruby_weapon-ex.js

```diff
@@ -305,6 +440,46 @@ export default {
         },
       },
     },
+    {
+      id: 'RubyEx White Agony Tether',
+      // White Agony is the blue head.
+      // This trigger doesn't run for the initial tether because the add
+      // spawns with the tether, but will run if somebody dies.
+      netRegex: NetRegexes.tether({ source: 'White Agony', id: '0011' }),
+      condition: Conditions.targetIsYou(),
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          text: {
+            en: 'Blue Head (Go East)',
```

Comment:
```suggestion
            en: 'Blue Head (Go East)',
            ko: '파란색 (동쪽)',
```

Reply:
Does it make sense to not include the directions here, since you can move the adds?

Reply:
You can ... technically move the adds, but they do a bunch of mechanics that lock them in place, and I have never seen them moved very far in any group, because they tether.  I would say these move much less than the HadesEx Igeyorhm and Lahabread adds in practice, and I would feel pretty comfy saying east and west there, personally.

The place where they might move is a failed swap, and this head tether mechanic comes out before then.  Arguably a tank could die, but it's probably a wipe if they both tether together on one tank at that point.

Mostly saying...I think the direction here is probably going to be correct and more helpful than just saying "go to red add".